Firestore/core/src/firebase/firestore/model/document_map.mm

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2018 Google
```

Comment:
Sorry for nitpicking, should this be 2019?

Reply:
Sorry. I should add a linter check for this.